src/content/topics/sdk/features/flag-changes.mdx

```diff
@@ -493,7 +493,7 @@ To subscribe to all feature flag changes, register listeners for change events f
 <CSTab label="JavaScript">
 
 ```js
-client.on('change', allChanges => {
+client.on('update', allChanges => {
```

Comment:
The node js client-side sdk uses the js common sdk so uses 'change' as well. Please also rollback these thank you.

Reply:
Updated